pennylane/math/quantum.py

```diff
@@ -1405,3 +1407,56 @@ def trace_distance(state0, state1, check_state=False, c_dtype="complex128"):
     eigvals = qml.math.abs(qml.math.eigvalsh(state0 - state1))
 
     return qml.math.sum(eigvals, axis=-1) / 2
+
+
+def expand_vector(vector, original_wires, expanded_wires):

```

Comment:
I think this might make more sense in the `matrix_manipulation` file.  It's basically the same thing as `expand_matrix`, just on a vector.